target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-ex62x0v2.dtsi

```diff
@@ -0,0 +1,395 @@
+/* Copyright (c) 2015, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2018, David Bauer <mail@david-bauer.net>
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+        model = "Netgear EX62x0v2";
+        compatible = "netgear,ex62x0v2";
+
+        soc {
+                rng@22000 {
```

Comment:
Use the existing labels for prng, mdio, crypto and watchdog instead.

Reply:
I don't fully follow what the ask is here, I did search to see if these labels are defined as something else where but I didn't get any additional context. Where would labels for these be defined so I can use the correct existing labels

Reply:
Ignore my previous comment, I see what you mean. I couldn't find the ipq4019.dtsi file that this dtsi points too in my local repo but googling for it, I was able to find the labels. Makes sense now